Frontend/kalvians_management_system-kms/src/Pages/Registration.jsx

```diff
@@ -1,9 +1,122 @@
-import React from 'react'
+import { Box, Button, TextField, Typography } from '@mui/material';
+import React from 'react';
+import { useState } from 'react';
+import { useNavigate } from 'react-router';
+
 
 const Registration = () => {
+
+  const [name,setName]=useState("");
+  const [email,setEmail]=useState("");
+  const [password,setPassword]=useState("");
+
+  const Navigate=useNavigate();
+
+  const handleRegistration=async()=>{
+
+   if(!name || !email || !password){
+      alert("Please fill all your details");
+      return;
+   }
+
+   if(!email.includes("@")){
+         alert("Please enter the correct E-mail address");
+         setEmail("")
+         return;
+   }
+
+   if(password.length>15 || password.length<8){
+      alert("Please enter the password between 8-15 characters");
+      setPassword("");
+      return;
+}
+
+   let data={
+      name,
+      email,
+      password
+   }
+
+      try {
+
+         let res = await fetch('http://localhost:3000/registration_data',{

```

Comment:
Add domain details `http://localhost:3000` to .env file and use process.env value here as well.